src/config/axios.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const axiosClient = axios.create({
+  baseURL: 'https://content-youtube.googleapis.com/youtube/v3/',
+});
+
+export default axiosClient;
```

Comment:
Good use of axios instance.

Go one more step ahead and try to put the baseURL value as Env variable, in order to  get this more easy to change on the time.